tests/src/utility/operations.ts

```diff
@@ -135,3 +138,50 @@ export async function handleConfirmationDialog(page: Page, dialogTitle = 'Confir
   const button = confirm ? dialog.getByRole('button', { name: 'Yes' }) : dialog.getByRole('button', { name: 'Cancel' });
   await button.click();
 }
+
+/**
+ * Async function that stops and deletes Podman Machine through Settings -> Resources page
+ * @param page playwright's page object
+ * @param machineVisibleName Name of the Podman Machine to delete
+ */
+export async function deletePodmanMachine(page: Page, machineVisibleName: string): Promise<void> {
+  const navigationBar: NavigationBar = new NavigationBar(page);
+  const dashboardPage: DashboardPage = await navigationBar.openDashboard();
+  await dashboardPage.mainPage.waitFor({ state: 'visible', timeout: 3000 });
```

Comment:
Please use await playExpect(object).tobeVisible() assertion to confirm screen navigation.

Reply:
I've been following the rest of the methods logic in the file, they don't seem to be using playExpect

Reply:
There are some pros on isVisible vs. waitFor. If it is not a problem. It s like 10 lines.

Reply:
done.